draft-lcurley-moq-probe.md

```
@@ -0,0 +1,215 @@
+
+
+# Padding
+Padding is optional and depends on the capabilities of the QUIC implementation.
+A publisher that does not support padding MUST still send PROBE_RESPONSE messages based on the actual sending rate.
+
+## QUIC-Level Padding
+The preferred method is for the QUIC implementation to send PING+PADDING frames.
+PADDING frames alone MUST NOT be used, as they are not ack-eliciting and can cause starvation of the congestion controller.
+PING+PADDING is transparent to the application and does not consume application-level flow control.
+
+## Datagram Padding
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)